src/runtime_src/tools/xclbinutil/FormattedOutput.cxx

```diff
@@ -909,6 +911,22 @@ reportKernels(std::ostream& _ostream,
       }
     }
   }
+  // Print all remaining IPs and the base address
+  for (auto& ptIPData : ipLayout) {
+    auto sKernelInstance = ptIPData.get<std::string>("m_name");
+    auto pos = sKernelInstance.find(':');
+    if (pos != std::string::npos){
+      auto sKernel = sKernelInstance.substr(0,pos);
+      auto sInstance = sKernelInstance.substr(pos+1);
+      if(std::find(kernel_names.begin(), kernel_names.end(), sKernel) == kernel_names.end()) {
+        _ostream << boost::format("%s %s\n") % "Kernel:" % sKernel;
+        _ostream << "--------------------------\n";
+        _ostream << boost::format("%-16s %s\n") % "Instance:" % sInstance;
+        auto sBaseAddress = ptIPData.get<std::string>("m_base_address");
+        _ostream << boost::format("   %-13s %s\n") % "Base Address:" % sBaseAddress;
+      }
+    }
```

Comment:
What happens when sKernel is not found in kernel_names?
If it needs to be handled, add code
if it can be ignored, add comment